python/neuroglancer/local_volume.py

```diff
@@ -83,8 +87,11 @@ def __init__(self,
         """
         super(LocalVolume, self).__init__()
         self.token = make_random_token()
-        self.data = data
-        self.shape = data.shape
+        if ts is not None and isinstance(data, ts.TensorStore):
+            self.data = TensorStoreLocalVolume(data)
+        else:
+            self.data = data
+        self.shape = self.data.shape
```

Comment:
Can you also add support for choosing a default `CoordinateSpace` from the additional data provided by TensorStore, namely `data.domain.labels`, and `data.dimension_units`?

Also can you add support for non-zero origin, e.g. set `voxel_offset = data.origin` and then set `data = data[ts.d[:].translate_to[0]]` ?

Finally can you add a few unit tests (just of LocalVolume directly perhaps)?